### ISSUE TITLE
Add 'Like/Approve' actions to a Comment Notification in addition to replying. 

### ISSUE THREAD

**Issue by mzorz** [wordpress-mobile/WordPress-Android#4523]
Comes from #4031 
We only have the "Reply" action. Adding Like / Approve actions to the notification for quick access to these actions.

**Comment by mzorz** [comment]
I've implemented this in 2 steps, one that only has the "LIKE" action and another one that also has the "Approve" action, but I first wanted to show how it looks so we decide which gets shipped in my PR:

### Like-only version
This one shows the "Like" addition:

![likeonly](https://cloud.githubusercontent.com/assets/6597771/18182319/df0428d6-7065-11e6-81cf-5a5a19c13b24.png)

Which seems pretty OK.

### Like-and-approve version
**But,** now if we add the  "Approve" action, this is how it looks like in portrait mode on my Moto X:

![likeapprove_portrait](https://cloud.githubusercontent.com/assets/6597771/18182347/f9371902-7065-11e6-8559-4c58696d789c.png)

And in landscape mode:

![screenshot_2016-09-01-16-50-33_mod](https://cloud.githubusercontent.com/assets/6597771/18182402/3f32a296-7066-11e6-8833-3a6935bdca6a.png)

This is not really what I expected, but there doesn't seem to be really that much space to add more actions though.

Some possible solutions:
a) ditch the 'Approve' option, leaving only `reply` and `like`
b) ditch the action texts, but I doubt it makes any difference (besides it'd remove the clarification each action title gives the icon itself) 

what are your thoughts? cc @hoverduck @mattmiklic

**Comment by hoverduck** [comment]
Does the Approve option appear on every notification, or only when the action is required?  Could the actions be changed so that if the comment needs approval then the options are "Approve" / "Spam" (or "Trash", if that's more frequently used...I'm not sure), and then if it doesn't need approval it's just "Reply" / "Like"?

**Comment by mzorz** [comment]
Great feedback @hoverduck ! That’d be great indeed - the thing is about what happens behind the scene. 
TL;DR we need to do processing in order to decide which buttons to show, and such processing might (or not) be a bit too much. We’d have to test (I'll do some of this tomorrow and see what I came come up with).

Long story: when a push notification is received by the OS and handled by the app, it performs some basic processing in order to build a visual notification (that is, what is actually seen by the user). For this case, we would need to check whether this particular comment is approvable by this user. 
Also, being that (only for now) all notifications are related to blogs that I’m an admin of, when you receive a new comment such a comment would likely be totally new and thus unapproved (that is, yet-to-be-approved). It follows we can safely assume this user can moderate (approve/trash/spam) this comment, leaving only the case where comments have been set to be published right away (in which case it wouldn’t make sense to have an “approve” button).

On the other hand, Reply/Like would be something you can always do, so I guess this might be the easiest solution in case everything else doesn't work
